app/views/payment-links/amount.njk

```diff
@@ -0,0 +1,64 @@
+{% extends "../layout.njk" %}
+
+{% block page_title %}
+Create a payment link - {{currentServiceName}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
+{% endblock %}
+
+{% block mainClasses %}fullwidth{% endblock %}
+
+{% block full_width_content %}
+<div class="grid-row">
+  <aside class="column-one-third">
+    <nav role="navigation" class="navigation settings-navigation">
+      <ul class="list">
+        <li class="active"><a href="{{ returnToStart }}">Create payment link</a></li>
+        <li><a href="{{ manage }}">Manage payment&nbsp;links</a></li>
+      </ul>
+    </nav>
+  </aside>
+  <section class="column-two-thirds">
+    {% include "includes/flash.njk" %}
+    <h1 class="page-title">Is the payment for a fixed amount?</h1>
+    <form action="{{ nextPage }}" class="form" method="post">
+      <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+      <div class="form-group{% if flash.genericError %} error{% endif %}">
+        {% if flash.errorType == 'paymentAmountType' %}
+        <span class="error-message">
+          Please choose an option
+        </span>
+        {% endif %}
+        <fieldset data-module="show-hide-content" id="fixed-or-variable">
+          <div class="multiple-choice" data-target="fixed-amount">
+            <input id="amount-type-fixed" type="radio" name="amount-type-group" value="fixed" {% if paymentLinkAmount or flash.errorType == 'paymentAmountFormat' %}checked{% endif %}>
```

Comment:
I see the logic for paymentAmountType is here, but would be better to encapsulate in the controller?